modules/videosipgw/JitsiVideoSIPGWSession.ts

```diff
@@ -6,18 +6,23 @@ import Listenable from '../util/Listenable';
 import * as VideoSIPGWConstants from './VideoSIPGWConstants';
 
 const logger = getLogger('modules/videosipgw/JitsiVideoSIPGWSession');
+import ChatRoom  from '../xmpp/ChatRoom';
```

Comment:
Imports should go at the top. Please rebase to make sure the linter picks it up.